adafruit_cursorcontrol/cursorcontrol_cursormanager.py

```diff
@@ -92,11 +99,12 @@ def is_clicked(self):
 
     def update(self):
         """Updates the cursor object."""
-        pressed = self._pad.get_pressed()
-        self._check_cursor_movement(pressed)
+        event = self._pad.events.get()
```

Comment:
Have you considered using ``get_into`` to avoid memory allocations?

Reply:
Ah shoot, I meant to change that before submitting thank you!